aodntools/timeseries_products/velocity_aggregated_timeseries.py

```diff
@@ -0,0 +1,316 @@
+import os
+import sys
+import netCDF4 as nc4
+import numpy as np
+import json
+from datetime import datetime
+import argparse
+
+import xarray as xr
+
+import aggregated_timeseries as TStools
+
+TEMPLATE_JSON = 'velocity_aggregated_timeseries_template.json'
+
+
+def sort_files(files_to_agg):
+    """
+    sort list of files according to deployment date
+    requires netcdf4 as nc4, dateutil.parser as parse
+    :param files_to_agg: List of files to sort
+    :return: sorted list of files
+    """
+
+    time_start = []
+    for file in files_to_agg:
+        with nc4.Dataset(file, 'r') as ds:
+            time_start.append(np.datetime64(ds.time_deployment_start))
+    tuples = sorted(zip(time_start, files_to_agg))
+    return [t[1] for t in tuples]
```

Comment:
1. change ```files_to_agg``` to ```files``` or file_list or file_str_list -> there is no agg here and put the type intention.
2. Change file to filestr in the loop -> again type intention.
2. ```return [file for _,file in tuples]``` is clearer

Reply:
changed